jsonLoader.go

```diff
@@ -252,12 +259,13 @@ func (l *jsonBytesLoader) LoaderFactory() JSONLoaderFactory {
 	return &DefaultJSONLoaderFactory{}
 }
 
+// NewStringLoader creates a new JSONLoader, taking a `[]byte` as source
```

Comment:
This should be `NewBytesLoader`

Reply:
Good catch! I definitely started to loose inspiration after a while 😂